js/runner.go

```diff
@@ -223,7 +223,7 @@ func (r *Runner) Setup(ctx context.Context, out chan<- stats.SampleContainer) er
 
 	v, err := r.runPart(setupCtx, out, stageSetup, nil)
 	if err != nil {
-		return errors.Wrap(err, stageSetup)
```

Comment:
I expect that this helps with printing the stack ?

Reply:
not really, printing of the stack is a separate issue, this just removes an extra duplicate `setup: ` from the error message that wasn't present in `teardown()` error messages